src/read-only-handler.ts

```diff
@@ -85,16 +83,20 @@ export class ReadOnlyHandler {
             return desc;
         }
         const shadowDescriptor = getOwnPropertyDescriptor(shadowTarget, key);
-        if (!desc.configurable && !shadowDescriptor) {
+        if (!isUndefined(shadowDescriptor)) {
+            return shadowDescriptor;
+        }
+        desc = wrapDescriptor(membrane, desc, getReadOnlyValue);
+        delete desc.set; // readOnly membrane does not allow setters
```

Comment:
this logic is pretty much the same as the other file except for this line. in readonly membrane, we never expose a setter.